lib/pymedphys/tests/e2e/cypress/integration/streamlit/metersetmap.spec.js

```diff
@@ -133,32 +130,17 @@ describe("When running all calculations", () => {
     cy.get(".stSelectbox").first().type("DICOM RTPlan file upload{enter}")
     cy.compute()
 
-    cy
-      .contains("DICOM import method")
-      .parent()
-      .find("input")
-      .last()
-      .click({ force: true });
-    cy.compute()
-
-    cy
-      .contains("Monaco Export Location")
-      .parent()
-      .find("input")
-      .last()
-      .click({ force: true });
-    cy.compute()
+    cy.radio("DICOM import method", "Search Monaco")
+    cy.radio("Monaco Export Location", "BLUE")
```

Comment:
I refactored out the radio button logic and made it be based on the radio button text content instead of the radio button order.